src/features/figma/components/FetchedSVG.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { memo, useEffect } from "react";
+import SVG from "react-inlinesvg";
+import { useDispatch } from "react-redux";
+
+import {
+  getRelativePosition,
+  getTreeHierarchyFromDOM,
+  hashCode,
+  registerHoverEventsOnFigmaEls,
+  REMOTE_PARENT_SVG_ID,
+} from "../utils";
+import { colorTheme } from "../../../theme";
+import {
+  defaultFigmaZone,
+  zonesSetTree,
+  zonesUpdatedByElementId,
+} from "../../zones/zonesSlice";
+
+export const getSvgUrlFromCurrentUrl = () => {
+  // get current url
+  const url = window.location.href;
+  // find in the url bytesimBucket, region and key
+  const bytesimBucket = url.match(/bytesimBucket=([^&]*)/)?.[1];
+  const region = url.match(/region=([^&]*)/)?.[1];
+  const key = url.match(/key=([^&]*)/)?.[1];
+
+  // if some of the object are not defined return the default test url
+  const defaultUrl =
+    "https://bytesim-bucket.s3.eu-west-3.amazonaws.com/1350067722696166.5_bytesimname_bytesim_dev___Page_1.svg";
+  if (!bytesimBucket || !region || !key) return defaultUrl;
```

Comment:
please add a TODO to display a warning instead of using the defaukt url after tests